src/ImageService.Data/ImageMessageRepository.cs

```diff
@@ -18,7 +17,7 @@ public ImageMessageRepository(IDataProvider provider)
 
     public List<Guid> Create(List<DbImageMessage> imagesMessages)
     {
-      if (imagesMessages.Contains(null))
+      if (imagesMessages == null || !imagesMessages.Any())
```

Comment:
if imagesMessages doesn`t have elements - then return null.  What happened If in collection will be only one element is Null?

Reply:
It will add an item with fields null.

Reply:
is it right behaviour?

Reply:
Check that there is no null element, and if there is one, then excluding them may turn out to be worse than just adding such an element to the database. Moreover, it will take up little space.

Reply:
if you have 2 elements of null and try to add it to db then you create element with id=0 and then when you tried to add second null element - you crashed. Is is valid case?